src/pages/home/Home.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect } from "react";
+import { useFilters } from "../../contexts/filters";
+import { Link, useNavigate } from "react-router-dom";
+import axios from "axios";
+import "./Home.css";
+const Home = () => {
+  const navigate = useNavigate();
+  const { filterState, filterDispatch } = useFilters();
+
+  useEffect(() => {
+    (async () => {
```

Comment:
suggestion: you can make a separate folder for api call so that your code looks clean